docs/advanced/Instruction-Document.md

```diff
@@ -0,0 +1,143 @@
+# KubeEdge Image PrePull Feature Guide Document
```

Comment:
1. Rename the document name, such as image-prepull.md. 
2. file name should not use capital letter
3. add a head for docs, ref to other document md.